### ISSUE TITLE
🐛EACCES error when running on WSL

### ISSUE THREAD

**Issue by jankowiakdawid** [rome/tools#2331]
### Environment information

```bash
laptop: Microsoft Surface Book 2
os: Windows 10 x64 [Microsoft Windows [Version 10.0.19043.1586]]
wsl2: Ubuntu-20.04
```

### What happened?

Running `npx rome@next format package.json`
returned error
```
❯ npx rome@next format package.json
/home/djankowiak/projects/review-backend/lib/node_modules/rome/bin/rome:28
                throw result.error;
                ^

Error: spawnSync /home/djankowiak/projects/review-backend/lib/node_modules/@rometools/cli-linux-x64/rome EACCES
    at Object.spawnSync (node:internal/child_process:1086:20)
    at Object.spawnSync (node:child_process:693:24)
    at Object.<anonymous> (/home/djankowiak/projects/review-backend/lib/node_modules/rome/bin/rome:21:47)
    at Module._compile (node:internal/modules/cjs/loader:1109:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
    at Module.load (node:internal/modules/cjs/loader:989:32)
    at Function.Module._load (node:internal/modules/cjs/loader:829:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47 {
  errno: -13,
  code: 'EACCES',
  syscall: 'spawnSync /home/djankowiak/projects/review-backend/lib/node_modules/@rometools/cli-linux-x64/rome',
  path: '/home/djankowiak/projects/review-backend/lib/node_modules/@rometools/cli-linux-x64/rome',
  spawnargs: [ 'format', 'package.json' ]
```

After installing it `npm i -D rome@next` (which worked) I'd tried `npx rome format package.json` with the same result.
I'd also tried adding it to scripts, but the result was the same.

### Expected result

It will run without error.

### Code of Conduct

- [X] I agree to follow Rome's Code of Conduct

**Comment by ematipico** [comment]
@jankowiakdawid have you tried formatting something that is not `.json`? Our formatter doesn't support JSON yet: https://rome.tools/#language-support

**Comment by MichaReiser** [comment]
This isn't WSL related but also happens on linux. Setting the executable permission on the binary solves the issue. I guess we can either set the executable permission when running the binary or during the installation.

**Comment by MichaReiser** [comment]
@leops do you know how we could fix this. I think we should address this before the release.

The esbuild setup is interesting. Their binary is in the `/bin/esbuild` so you can run it with `npx esbuild`. However, they depend on a post install script.

**Comment by leops** [comment]
I've been meaning to add a postinstall script to show a warning on unsupported platforms anyway, so I could also perform a file copy + chmod there

**Comment by ematipico** [comment]
@leops Can we close this issue now?

**Comment by leops** [comment]
I still would want to release a new version of the npm package and test the full installation + execution process to be sure, but I guess we can also close it and re-open if it's not properly fixed

**Comment by leops** [comment]
Closing as a fix was merged in #2333 and published in `0.4.2`
